tree/tree/src/TTreeCache.cxx

```diff
@@ -240,8 +251,22 @@ of effective system reads for a given file with a code like
 #include "TLeaf.h"
 #include "TFriendElement.h"
 #include "TFile.h"
+#include "TMath.h"
 #include <limits.h>
 
+// TODO: Copied from TBranch.cxx
```

Comment:
:) Even the TODO is in three files now.

Is `RConfig.h` part of the public API?  If so, do we want to put this into the public API?

I'll prepare a separate PR.